src/Batch/Batch/Az.Batch.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '2.0.0'
+ModuleVersion = '2.0.1'
```

Comment:
This was a breaking change release for Az.Batch

Reply:
Yes, the preview version is 2.0.0-preview, so I manually change to 2.0.1

Reply:
Yeah, my guess is that the mismatch is what prevented the update of az.psd1 and for recovery services